src/App.tsx

```diff
@@ -61,8 +62,15 @@ export default class App extends React.Component<IAppProps> {
     }
 
     public render() {
-        const platform = global && global.process ? global.process.platform : "web";
 
+        const platform = global && global.process ? global.process.platform : "web";
+        if(!this.props.auth.remember){
+            window.addEventListener("beforeunload", (e) => {
+                event.preventDefault();
+                localStorage.removeItem("token");
+            }
+            )
```

Comment:
Formatting is a bit off here. I think integrating Prettier to the project might be a good idea.